src/dev/build/tasks/build_kibana_platform_plugins.ts

```diff
@@ -27,7 +27,7 @@ export const BuildKibanaPlatformPlugins: Task = {
       repoRoot: REPO_ROOT,
       outputRoot: build.resolvePath(),
       cache: false,
-      oss: build.isOss(),
+      oss: false,
```

Comment:
We can just remove this line, `oss: false` has always been confusing and it's the default anyway.